src/middleware/__tests__/processJwt.int.test.ts

```diff
@@ -49,6 +49,23 @@ describe('processJwt', () => {
 		processJwt(mockRequest, mockResponse, nextMock);
 	});
 
+	it('does NOT populate an auth value when an auth header with an invalid issuer is sent', (done) => {
+		const mockJwt = getMockJwt({
+			iss: 'NotTheCorrectIssuer',
```

Comment:
@bickelj I am realizing this is just one edge case, but probably doesn't account for a situation where the `iss` is what we expect but the token itself was created with an invalid cert.

I'm not entirely sure how to mock that case, though.  Is that something you know how to do off hand?  If not, I can look into figuring it!

Reply:
@slifty Off hand I suppose we'd generate two issuers, one valid and one invalid, and two tokens, one from each, then replace the invalid token's attributes selectively from the valid one. The signature check should fail. I don't know off the top of my head how hard that will be, though. Looking.

Reply:
@slifty And that may not be what you are asking. I think you were suggesting that the issuer is as expected but the signing key is not as expected and that the JWT is valid with respect to the actual (unexpected) signing key? If so, my above suggestion probably wouldn't work because it would require a different issuer URL to get the mock JWKS library to generate a different key. Hmm.

Reply:
It took a bit to come back to this context. Perhaps this needs a better overview in the tests' documentation or something.

We use `mockJwks`, a stateful server, from `src/test/mockJwks.ts`, to intercept requests to its given issuer URL. The `mockJwks` server is started in `src/test/integrationSuiteSetup.ts` in `beforeEach`. Perhaps inside a test you described above, a second `badMockJwks` could be created using the same arguments, same issuer, etc., and hopefully would create a new key (we'd have to verify it didn't cache and return the same server), start that second server, create an identical-except-signed-with-badMockJwks-key JWT, stop that server, then attempt to pass this JWT on. Does that sound like the right approach to you?